layouts/404.html

```diff
@@ -0,0 +1,54 @@
+{{ define "main" }}
+<section class="hero is-medium has-background-image">
+  <div class="hero-body">
+    <div class="container has-text-centered">
+      <p class="title is-size-1 is-size-2-mobile has-text-weight-bold">
+        <span class="has-text-white">
+          404
```

Comment:
I generally love terse pages, but maybe a bit more text to this title (possibly as a subtitle), might be good. :) Even if it is only "Page not found"